src/thor/maestro/jenkins.py

```diff
@@ -65,14 +79,40 @@ def check_result_of_job(self, job_name, expected_release_version):
                 f"The release version of latest job is {release_version} while the expected version is {expected_release_version}"
             )
 
-    # TODO: store the task result to database
+    def write_task_result(self, job_name, expected_release_version):
+        """ Uses check_result_of_job to ... check the result of the job. 
+        Uses parameters given above (self, str job_name, str expected_r_v. 
+        Once result is gotten, uses task_dao's create_task and 
+        release_dao's release_id_lookup to create an appropriate task. 
+        
+        Note: If a job with the same name and release_id already 
+        exists within the database, we update the result in-place
+        instead of creating a new Task. """
+
+        result = self.check_result_of_job(job_name, expected_release_version)
+        lu = release_id_lookup_class()
```

Comment:
release_id_lookup_class is manually replaced by the patch with returnThree, which is defined within the test_jjm_write_task_result file itself. This is to prevent errors with release_id_lookup from spreading and affecting the unit test write_no_prior, which should only examine the behavior of write_task_result. 

release_id_lookup_class is imported from dao/release_dao, where it serves as a helper class to allow release_id_lookup to be mocked by pytest-mock. 

Variable name lu will be changed, it's a holdover from when release_id_lookup was just look_upper.